mcp/gcp_ae.tf

```diff
@@ -39,7 +39,7 @@ resource "google_storage_bucket" "self" {
   default_event_based_hold = false
   labels = lookup(local.project, "labels", null)
   # ToDo handle region better between app engine and cloud storage
-  location = "${local.gae.location_id}1"
+  location = local.gae.location_id
```

Comment:
Organisational policy allows for europe-west2 locations. So changed this to match the location specified in app engine, since specifying `europe-west2` in `gcp_ae.yml` would change to `europe-west21`